Sprint-2/stretch/till.js

```diff
@@ -5,13 +5,23 @@
 // Then it should return the total amount in pounds
 
 function totalTill(till) {
+
+  const tillCoins = {
+    "1p": 1,
+    "5p": 5,
+    "50p": 50,
+    "20p": 20,
+  };
```

Comment:
Can you think of a better name for this array (since you are using it as a lookup table or map)?